src/components/LoginScreen.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import LoginWithSlackButton from "./LoginWithSlackButton";
+import styled from 'styled-components'
+import Colors from "../constants/Colors";
+import Instructions from "../components/Instructions";
+
+class LoginScreen extends Component {
+    render () {
+        return (
+            <Container>
+                <WelcomeText>Kudos for coming!</WelcomeText>
+                <IntroText>
+                  This app helps to show our thankfulness.
+                </IntroText>
+                <LoginWithSlackButton/>
+                <Description>We&#39;re happy you're here! <span role="img" aria-label="Hug emoji">🤗</span></Description>
```

Comment:
this second quote breaks here formating on github :(

Reply:
GitHub sucks. Let's more to exchanging .rar.